overrides/cf.d.ts

```diff
@@ -0,0 +1,302 @@
+// TODO: add Response cf object too
```

Comment:
Oh I missed this. What should we do for the Response type?

Reply:
Sorry I misread this, I thought this was about the Response class itself, which was hidden in another file because github had collapsed it. All good.